docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -0,0 +1,399 @@
+Elastic Observability.
+
+.Advanced architecture of CI/CD Observability with Elastic Observability
+image::images/advanced-arch-observability.png[Advanced architecture of CI/CD Observability]
+
+[discrete]
+[[ci-cd-administrators]]
+== Observability for CI/CD administrators
+
+Elastic Observability allows CI/CD administrators to monitor and troubleshoot CI/CD
+platforms and detect anomalies.
+
+[discrete]
+[[ci-cd-monitoring]]
+=== CI/CD platform monitoring and alerting
+
+Elastic Observability helps CI/CD administrators monitor their platform by providing KPI dashboards
+of CI systems.
+
+The Jenkins health dashboards provide insights on the build executions, the failures, the
+provisioning of build agents, the active and idle workers, or the JVM health.
+
+[role="screenshot"]
+.Jenkins KPIs in Elastic Observability
+image::images/ci-cd-overview.png[CI/CD overview]
+
+[role="screenshot"]
+.Jenkins Provisioning KPIs in Elastic Observability
+image::images/jenkins-kpis.png[Jenkins KPIs]
+
+[role="screenshot"]
+.Jenkins JVM health indicators in Elastic Observability
+image::images/jenkins-jvm-indicators.png[Jenkins JVM health indicators]
+
+[discrete]
+[[ci-cd-troubleshooting]]
+=== CI/CD platform troubleshooting
+
+CI/CD administrators need to assess the impact of anomalies when troubleshooting platform problems quickly,
+whether troubleshooting just one pipeline to much broader outages impacting many pipelines or the entire CI/CD platform.
+
+Elastic Observability enables troubleshooting CI platform outages by providing visualizations of pipeline
+executions as distributed traces, along with the capability to slice and dice pipeline executions in any dimension
+to assess the nature and the impact of the outage.
+
+In the following image, a Jenkins CI build failed, and its exceptions are reported as errors.
+Select any of those errors to view the specific information. In this case, it's errors relating to
+the CI agent that stopped.
+
+[role="screenshot"]
+.Jenkins pipeline build error in Elastic Observability
+image::images/jenkins-pipeline-build.png[Jenkins pipeline builds]
+
+The Errors overview screen provides a high-level view of the exceptions that CI builds catch.
+Similar errors are grouped to quickly see which ones are affecting your services
+and allow you to take action to rectify them.
+
+[role="screenshot"]
+.Jenkins jobs and pipelines errors in Elastic Observability
+image::images/jenkins-pipeline-errors.png[Jenkins pipeline build errors]
+
+[role="screenshot"]
+.Concourse CI pipeline execution as a trace in Elastic Observability
+image::images/concourse-ci-traces.png[Concourse CI traces view]
+
+[discrete]
+[[ci-cd-developers]]
+== Observability for developers
+
+Development teams need to continuously optimize their ever-changing CI/CD pipelines to improve
+their reliability while chasing for faster pipelines. Visualizations of pipelines as distributed
```

Comment:
This is very awkwardly phrased. Perhaps the word `for` is unintentional?

Reply:
Yes, great spot. Thanks!